audience_manager/public_api/definitions/attributes_data.yaml

```diff
@@ -40,13 +40,28 @@ properties:
     description: Attributes of the occurred event.
     items:
       type: object
+      required:
+        - name
+        - value
```

Comment:
It's not required when action is `add`. Maybe we should resign from that default to simplify API?

Reply:
Hidden feature :stuck_out_tongue: